Dockerfile

```diff
@@ -15,10 +15,19 @@ ENV PYTHONUNBUFFERED=1
 
 COPY --from=builder /app .
 
+# Install dependencies
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
+# Set environment variables for sensitive data
+ENV DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY}
+ENV DB_HOST=${DB_HOST}
+ENV DB_NAME=${DB_NAME}
+ENV DB_USER=${DB_USER}
+ENV DB_PASSWORD=${DB_PASSWORD}
```

Comment:
Avoid using environment variables directly in the Dockerfile for sensitive data like `DJANGO_SECRET_KEY`, `DB_HOST`, `DB_NAME`, `DB_USER`, and `DB_PASSWORD`. Consider using Docker secrets or environment variables passed at runtime to manage these securely.